webapp/lib/view.dart

```diff
@@ -815,6 +815,13 @@ class ConversationListPanelView {
   Map<String, ConversationSummary> _phoneToConversations = {};
   ConversationSummary activeConversation;
 
+  num _totalConversations = 0;
+  void set totalConversations(num v) {
```

Comment:
Should `num` be changed to `int`? Can or should this ever be a non integer?

Reply:
good point - yes, it should be an int. Fixed now, thanks!